src/pages/workspace/WorkspaceCard.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {View, ScrollView, Image} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import styles from '../../styles/styles';
+import ONYXKEYS from '../../ONYXKEYS';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import Text from '../../components/Text';
+import Button from '../../components/Button';
+import variables from '../../styles/variables';
+import themeDefault from '../../styles/themes/default';
+import ROUTES from '../../ROUTES';
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** The details about the user that is signed in */
+    user: PropTypes.shape({
+        /** Whether or not the user is subscribed to news updates */
+        isFromPublicDomain: PropTypes.bool,
+    }),
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    user: {
+        isFromPublicDomain: false,
+    },
+};
+
+const WorkspaceCard = ({
+    user,
+    translate,
+}) => {
+    /**
+     * When user is on public domain, navigate the user to Add new bank Account.
+     *
+     */
+    const getStarted = () => {
+        if (user.isFromPublicDomain) {
+            Navigation.navigate(ROUTES.BANK_ACCOUNT_NEW);
+        }
+    };
```

Comment:
NAB, I think we can just have this be inline in the `onPress` callback since they are only used once.